spec/requests/other_incomes_spec.rb

```diff
@@ -10,7 +10,7 @@
 
     subject { post assessment_other_incomes_path(assessment_id), params: params.to_json, headers: headers }
 
-    UUID_REGEX = /^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$/.freeze
+    uuid_regex = /^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$/.freeze
```

Comment:
Convention is to have ruby contstants in all caps, so a better option here might be to move the contsant outside the block (eg line 2) or disable the cop as in `test_runner_spec.rb `.